docs/src/network.md

```diff
@@ -826,6 +848,25 @@ page.route_from_har("example.har")
 await context.RouteFromHARAsync("example.har");
 ```
 
+### Mocking with HAR
+
+You can modify the response from the HAR file and then run your tests against the modified response.
```

Comment:
Not sure how the snippet below is different from the previous sections, nor how does it highlight "you can modify the response" part?

Reply:
yeh I need to add that they need to modify the .txt file to modify the response, will fix